src/libcommonserver/io/iohelper.cpp

```diff
@@ -514,6 +514,20 @@ bool IoHelper::tempDirectoryPath(SyncPath &directoryPath, IoError &ioError) noex
 }
 
 bool IoHelper::logDirectoryPath(SyncPath &directoryPath, IoError &ioError) noexcept {
+    //!\ Don't use IoHelper::logger() here, as it Log::instance() may not be initialized yet. /!\_
```

Comment:
```suggestion
    //!\ Don't use IoHelper::logger() here, as its Log::instance() may not be initialized yet. /!\
```

Reply:
The line can't end with a backslash, it is why i put the underscore at the end.

Reply:
[57fffe2](https://github.com/Infomaniak/desktop-kDrive/pull/159/commits/57fffe2c590b05105eb666ffda35c14fc6e3c640)